packages/loader/container-loader/src/loader.ts

```diff
@@ -452,6 +452,19 @@ export class Loader implements IHostLoader {
 			request.headers[LoaderHeader.cache] !== false &&
 			pendingLocalState === undefined;
 		const fromSequenceNumber = request.headers[LoaderHeader.sequenceNumber] ?? -1;
```

Comment:
Here would be nice to stop using -1 and instead explicitly handle undefined also.